perl/p5-tcl-ptk/Portfile

```diff
@@ -47,6 +42,6 @@ if {${perl5.major} != ""} {
 
     # Bypass the test-for-tk since it doesn't work from MacPorts
     # (Perl never gets any output from tclsh for some reason)
-    patchfiles-append \
-                    patch-Makefile.PL.diff
+    configure_args-append \
```

Comment:
This causes a Portfile error:
`invalid command name "configure_args-append"`

This should instead be:
`configure.args-append`

Reply:
Whoops, `port` did tell me to fix that, so after thinking I fixed it, I forgot to stage the fix in my commit. Fixed in 81ae832c39, thanks for catching that.